generators/chipyard/src/main/scala/config/AbstractConfig.scala

```diff
@@ -17,6 +17,7 @@ class AbstractConfig extends Config(
   new chipyard.config.WithBootROM ++                             // use default bootrom
   new chipyard.config.WithUART ++                                // add a UART
   new chipyard.config.WithL2TLBs(1024) ++                        // use L2 TLBs
+  new chipyard.config.WithNoSubsystemDrivenClocks ++             // drive the subsystem diplomatic clocks from ChipTop instead of using implicit clocks
```

Comment:
Glad we an abstract config 👍 
I would be tempted to make the default config the divided version, since that is more "realistic".

Reply:
That would be cool, if only it worked :cry:

Reply:
I think we should do that once it does work.